.github/workflows/bach-tests.yml

```
@@ -61,11 +68,27 @@ jobs:
         run: |
           cd bach
           pycodestyle bach sql_models
-      - name: Unit tests
+      - name: Unit tests postgres
         run: |
           cd bach
-          pytest tests/unit/ -W error::DeprecationWarning
-      - name: Functional tests
+          pytest -n 4 --dist loadgroup --postgres tests/unit
```

```suggestion
          pytest --postgres tests/unit
```
is actually slightly (0.5 seconds) faster on my machine, as the overhead of spinning up the workers is more than the time gained from the parallelization

---

same for the unittests below

---

ran a test, but on GH the test went from 5mins to 6 mins (per python version). So in total 3 minutes slower, so reverted the change back to 4 processes.

---

I think the test was flawed. What took so much longer were the functional tests, not the unit tests.

---

But, as this is about seconds, and we have been chatting about this for many many seconds. Let's just merge this as!

---

oh sure, was just waiting for the tests to be green after the last change!